content/en/certificates.md

```diff
@@ -3,78 +3,102 @@ title: Chain of Trust
 ---
 
 {{< lastmod >}}
 
 # Root Certificates
 
 Our roots are kept safely offline. We issue end-entity certificates to subscribers from the intermediates in the next section.
+For additional compatibility as we submit our new Root X2 to various root programs, we have also cross-signed it from Root X1.
 
 * Active
-  * [ISRG Root X1 (self-signed)](/certs/isrgrootx1.pem.txt)
+  * ISRG Root X1 (RSA 4096)
+    * [self-signed](/certs/isrgrootx1.pem.txt)
+  * ISRG Root X2 (ECDSA P-384)
+    * [self-signed](/certs/isrg-root-x2.pem.txt)
+    * [cross-signed by ISRG Root X1](/certs/isrg-root-x2-cross-signed.pem.txt)
 
 We've set up websites to test certificates chaining to our roots.
 
-* ISRG Root X1 Valid Certificate
-  * [https://valid-isrgrootx1.letsencrypt.org/](https://valid-isrgrootx1.letsencrypt.org/)
-* ISRG Root X1 Revoked Certificate
-  * [https://revoked-isrgrootx1.letsencrypt.org/](https://revoked-isrgrootx1.letsencrypt.org/)
-* ISRG Root X1 Expired Certificate
-  * [https://expired-isrgrootx1.letsencrypt.org/](https://expired-isrgrootx1.letsencrypt.org/)
+* ISRG Root X1
+  * [Valid](https://valid-isrgrootx1.letsencrypt.org/)
+  * [Revoked](https://revoked-isrgrootx1.letsencrypt.org/)
+  * [Expired](https://expired-isrgrootx1.letsencrypt.org/)
+* ISRG Root X2
+  * [Valid](https://valid-isrgrootx2.letsencrypt.org/)
+  * [Revoked](https://revoked-isrgrootx2.letsencrypt.org/)
+  * [Expired](https://expired-isrgrootx2.letsencrypt.org/)
 
 # Intermediate Certificates
 
-Under normal circumstances, certificates issued by Let’s Encrypt will come from “Let’s Encrypt Authority X3”. The other intermediate, “Let’s Encrypt Authority X4”, is reserved for disaster recovery and will only be used should we lose the ability to issue with “Let’s Encrypt Authority X3”. We do not use the X1 and X2 intermediates any more.
+Under normal circumstances, certificates issued by Let's Encrypt will come from "Let's Encrypt Authority X3".
+We have also issued new intermediates ("Lets Encrypt E2" and "Let's Encrypt R3"), which we will begin issuing from soon. This page will be updated when that transition is complete.
+Our other intermediates ("Let's Encrypt Authority X4", "Let's Encrypt E2", and "Let's Encrypt R4") are reserved for disaster recovery and will only be used should we lose the ability to issue with our primary intermediates.
+We do not use the X1 and X2 intermediates any more.
 
-IdenTrust has cross-signed our intermediates for additional compatibility.
+IdenTrust has cross-signed our RSA intermediates for additional compatibility.
 
 * Active
-  * [Let's Encrypt Authority X3 (IdenTrust cross-signed)](/certs/lets-encrypt-x3-cross-signed.pem.txt)
-    * [Let's Encrypt Authority X3 (Signed by ISRG Root X1)](/certs/letsencryptauthorityx3.pem.txt)
+  * Let's Encrypt Authority X3
+    * [signed by ISRG Root X1](/certs/letsencryptauthorityx3.pem.txt)
+    * [cross-signed by IdenTrust](/certs/lets-encrypt-x3-cross-signed.pem.txt)
+* Upcoming
+  * Let's Encrypt R3
```

Comment:
This is a bit confusing since the string "Let's Encrypt R3" doesn't actually appear in any of our certificates. Let's include the separate O field in this and others for maximum clarity:

```
O = Let's Encrypt, CN = Let's Encrypt Authority X3
O = Let's Encrypt, CN = R3
```

etc

Reply:
Done, I've added algorithm and O/CN for all roots and intermediates.